src/sas_bluesky/panda_gui.py

```diff
@@ -562,4 +540,5 @@ def build_pulse_frame(self):
 
     # dir_path = os.path.dirname(os.path.realpath(__file__))
     # config_filepath = os.path.join(dir_path, "profile_yamls", "panda_config.yaml")
-    PandAGUI(configuration=BL_PROF.DEFAULT_EXPERIMENT)
+    instrument_session = input("Instrument Session/Visit?")
+    PandAGUI(configuration=experiment_profile(BL, instrument_session))
```

Comment:
How you're getting the instrument_session/visit from the `_root` of the PathProvider in dodal is about to cause you a headache when we try and remove the PathProvider impl from dodal- it's a private member, you shouldn't rely on its existence. I figured prompting here was a good time?